src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,43 @@
+package racingcar;
+
+import java.util.Random;
+
+public class Car {
+	public static final int LIMIT_RANDOM_NUMBER = 10;
+	public static final int MOVE_CONDITION = 4;
+	public static final String movingUnit = "-";
```

Comment:
```suggestion
	public static final String MOVING_UNIT = "-";
```

String 상수 또한, Int 상수 처럼 네이밍해주시면 좋을 것 같습니다.